src/gui/plugins/visualize_contacts/VisualizeContacts.hh

```diff
@@ -20,6 +20,9 @@
 
 #include <memory>
 
+#include <ignition/msgs/boolean.pb.h>
+#include <ignition/msgs/stringmsg.pb.h>
```

Comment:
I think this will create some warnings on Windows

```suggestion
#ifdef _MSC_VER
#pragma warning(push, 0)
#endif
#include <ignition/msgs/boolean.pb.h>
#include <ignition/msgs/stringmsg.pb.h>
#ifdef _MSC_VER
#pragma warning(pop)
#endif
```

Reply:
Fixed in [`76a7154`](https://github.com/ignitionrobotics/ign-gazebo/pull/907/commits/76a715450d59016d893b0eb657b489562e8acfb7)